app/src/main/java/com/projectit3048c/dao/IFoodDAO.kt

```diff
@@ -3,9 +3,8 @@ package com.projectit3048c.dao
 import com.projectit3048c.dto.Food
 import retrofit2.Call
 import retrofit2.http.GET
-import retrofit2.http.Query
 
-interface iFoodDAO {
+interface IFoodDAO {
```

Comment:
"I" in an interface name should be capitalized

Reply:
Yes.  UpperCamelCase is the proper naming convention for interfaces and classes.